docs/en/toolkit/learning-resources/guides/dynamic_filesystem_configuration.md

```diff
@@ -0,0 +1,398 @@
+---
```

Comment:
some kind of merge conflict thing that this file shows up as added in this PR?

Reply:
Test-merged locally and got the expected result...  I believe this is just a glitch in the display here.